src/roiextractors/extraction_tools.py

```diff
@@ -480,3 +487,59 @@ def todict(matobj):
         else:
             dict[strg] = elem
     return dict
+
+
+def get_package(
+    package_name: str,
+    installation_source: str = "pip",
+    package_installation_display: Optional[str] = None,
```

Comment:
I would for `name` instead of `display` here (or something else).

Reply:
Thought here (though this can apply to `installation_source`) was that it could be a string that also has other information other than just the name. Example

```
installation_source = "conda -c conda-forge"
```

or

```
package_installation_display = "--no-deps special-name-of-Package[special_options_too]"
```

Though IDK, it could also be clearer to add those other things as `installation_options: str = ""` separately and keep the other options as strictly the name mappings?

Reply:
My suggestion would be to use a single argument called `installations_instructions` for line 544 so you can get rid of `installation_source` and `installation_display`as they don't seem to lend themselves that well to that decomposition.

Also, would use `package_name` instead of `package_installation_display` on line 538.

Reply:
Done